crates/dbs-interrupt/src/notifier.rs

```diff
@@ -0,0 +1,216 @@
+// Copyright 2019 Alibaba Cloud. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
+
+//! Device event notifier to inject interrupt to guest.
+
+use std::io::Error;
+use std::sync::Arc;
+
+use downcast_rs::{impl_downcast, Downcast};
```

Comment:
we don't need to use `downcast_rs`, we can use fat pointer's downcast function directly

Reply:
To use the downcast function, you must add functions such as `as_any`, which is what downcast_rs does,  I feel no need to implement it again. 

In the previous version, I tried to implement it myself. When using it, I needed to import the `AsAny` trait. The code is basically the same as the `downcast_rs` crate.

Reply:
I see, it's ok